README.rst

```diff
@@ -1,6 +1,7 @@
-.. image:: https://docs.pytest.org/en/stable/_static/pytest1.png
+.. image:: https://github.com/nicoddemus/pytest/raw/transparent-logo-8158/doc/en/img/pytest_logo_curves.svg
    :target: https://docs.pytest.org/en/stable/
    :align: center
+   :height: 200
```

Comment:
This seems to be working too so no need to edit SVG maybe.

Reply:
Yeah I didn't edit it. 👍

Reply:
Bummer `align: center` doesn't seem to work though... 😕

Reply:
Yep, GH support for RST is quite limited.

Reply:
If you are happy with the results, can you approve and then we can get this online? 😁